### ISSUE TITLE
Hardly to understand your image alignment from matlab codes

### ISSUE THREAD

**Issue by yiakwy** [AlfredXiangWu/face_verification_experiment#142]
since you used matlab to load something else, it is very hard for third parties to understand your image preprocessing and alignment. Please update the doc, that just provide us with a caffemodel is meaningless. LFW images consists of 3 channels why did you just use 1 channel (i.e. grey level) in your input model? Do you have a plausible excuse to support your claim?

Just providing a caffemodel is meaningless, if we don't understand what your are doing, we have to train our own model instead of using your model. If you really want us to use your model, please tell us:

1. step by step to preprocess images, because this deeply relates to your fine tuned parameters
2. in face recognition, we have three tasks: identity recognition, comparison,  and image feature vector. If you are doing the first task, you should tell use which dataset you used to train and how to test output labels on LFW. If you are doing the other tasks, you should tell how to use the model to do the other tasks so that your claim in papers is ture and real.

Please do not use matlab again. Most people don't want to pay to matlab IDE, it is hard for them to test whether you are right.

**Comment by medivhna** [comment]
MATLAB is not a sin.

If you don't like it, you can fork this project and transplant the code to any platform.

Your attitude is disgusting.

**Comment by yiakwy** [comment]
@medivhna  No, it is NOT about **ATTITUDE**, it is about **SCIENCE** and **RESEARCH COMPLETE** (you should 've known it in university as a graduate student). Don't make it personal. This is truth you didn't make it clear how you complete image face task:

1. what task you are doing on LFW , image comparison, identification or something else?
2. You hide your preprocessing in mat codes. It is a problem. You should be language independent.
3. Fork you project meaning I admire your work. You should know it already. 

What I need is 
1) a pretrained model, caffemodel is welcome, to  do prediction, hence you should  be good aware of what model output
2) what they output for LFW? image feature vector, or  labels probability.

Good I check your model that the output is softmax, because if it is a vector, I will use different strategies to detect an image which contains figure from LFW.

3) Since you output labels. It is your responsibility to tell reader unambiguously what is your image processing and how did your deal with labels! for example lft_attribute file

Since you claim you are good at LFW, I choose trust you. 

I am asking your do you want people to use your codes or not. Then answer reside in your head.

NOT ALL PEOPLE WILL TAKE TIME, to see your codes because most of papers are bullshit, even some famous ones.

**Comment by yiakwy** [comment]
@medivhna Tool is not sin, sin is from people.

If you really like to use it, please write matlab codes in matlab style and attached description accordingly.

Your words are ignorance.
